riff-raff/app/schedule/DeployJob.scala

```diff
@@ -61,7 +62,11 @@ object DeployJob extends Logging with LogAndSquashBehaviour {
     lastDeploy.state match {
       case RunState.Completed =>
         val params = extractDeployParameters(lastDeploy)
-        if (scheduledDeploysEnabled) {
+        if (
+          lastDeploy.parameters.selector.isInstanceOf[DeploymentKeysSelector]
+        ) {
+          Left(SkippedDueToPreviousPartialDeploy(lastDeploy))
```

Comment:
Not caused by this PR, but would it be okay to add a log line for the bit this hits on lines 44-45? To indicate a notification is sent for a failed scheduled deploy.

I.e. at this bit:

```
      case Left(error: ScheduledDeployNotificationError) =>
        scheduledDeployNotifier.scheduledDeployFailureNotification(error)
```

above. Shout if not obvious which bit I mean.

Reply:
done in https://github.com/guardian/riff-raff/commit/f6fa8dff5ee9982ffa3d52d0a5a31e4dde4f3505